sdk/communication/Azure.Communication.MediaComposition/src/MediaCompositionClient.cs

```diff
@@ -130,13 +130,13 @@ public virtual Response<MediaCompositionBody> Get(string mediaCompositionId, Can
         /// <param name="inputs"> The media inputs. </param>
         /// <param name="outputs"> The media outputs. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<MediaCompositionBody>> CreateAsync(string mediaCompositionId, MediaCompositionLayout layout, IDictionary<string, MediaInput> inputs, IDictionary<string, MediaOutput> outputs, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<MediaComposition>> CreateAsync(string mediaCompositionId, MediaCompositionLayout layout, IDictionary<string, MediaInput> inputs, IDictionary<string, MediaOutput> outputs, CancellationToken cancellationToken = default)
         {
             using var scope = _clientDiagnostics.CreateScope("MediaCompositionClient.Create");
             scope.Start();
             try
             {
-                return await RestClient.CreateAsync(mediaCompositionId, mediaCompositionId, layout, inputs, outputs, CompositionStreamState.NotStarted, cancellationToken).ConfigureAwait(false);
+                return await RestClient.CreateAsync(mediaCompositionId, mediaCompositionId, layout, inputs, outputs, new CompositionStreamState(StreamStatus.NotStarted), cancellationToken).ConfigureAwait(false);
```

Comment:
Does our service allow to set the StreamStatus? I think we should make that property read-only for clients and only settable via the :start, :stop POST actions

Reply:
Yea, I think you're right Dom. It is already set as a read-only property, and even if we do set on the client, it will not be respected. I can modify to null to avoid confusions